File: Week-1/Day-3/palindrome.py
word = input("Please type in a word: ")
reversed_word = ""

for index in range(len(word)-1,-1,-1):
    reversed_word += word[index]


if word == reversed_word:
    print(f"{word} is a Palindrome")
else:
    print(f"{word} is NOT a Palindrome")

File: Week-4/Day-1/js/palindrome.js
let word = "Mom".toLowerCase()
let reversedWord = ""

for (let i = word.length - 1; i >= 0; i--) {
    reversedWord += word[i]
}

function palindrome() {

    if (word == reversedWord) {
        return true
    }

    return false

}

console.log(palindrome())